index.html

```diff
@@ -30,5 +30,82 @@ <h2>I’am a Software Developer</h2>
         <div class="illustration"><img src="images/illustration-header-mobile.svg" alt="illustration"></div>
     </div>
 </section>
+<div class="works">
+    <div class="works_title">
+        <div class="title">My Recent Works</div>
```

Comment:
Kindly. Titles should be H1, H2, H3, etc. tags. (Depending on the importance on your site), this will make your site more accessible.